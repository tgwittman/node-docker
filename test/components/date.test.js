import React from 'react'
import renderer from 'react-test-renderer'
import Date from '../../src/components/date'

test('Date component renders correctly', () => {
    const tree = renderer.create(
        <Date dateString={'2021-01-01'}/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
})
