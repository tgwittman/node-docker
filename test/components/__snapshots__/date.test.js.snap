// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date component renders correctly 1`] = `
<time
  dateTime="2021-01-01"
>
  January 1, 2021
</time>
`;
